### PR TITLE
Fix syntax error

### PR DESCRIPTION
**Pull request by hezhizhen**

### CODE REVIEW

**Review by Aut0R3V — APPROVED** [review]

File: content/en/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ a new instance.
 The name of a Service object must be a valid
 [DNS label name](/docs/concepts/overview/working-with-objects/names#dns-label-names).
 
-For example, suppose you have a set of Pods that each listen on TCP port 9376
+For example, suppose you have a set of Pods that each listens on TCP port 9376
```
Comment (kbhawkey) [inline content/en/docs/concepts/services-networking/service.md:77]: Hi @hezhizhen .
What do you think about updating the sentence to:
For example, suppose you have a set of Pods where each Pod listens on TCP port 9376 and contains the
label `app=MyApp`:

Comment (hezhizhen) [reply content/en/docs/concepts/services-networking/service.md]: @kbhawkey What about now?

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-25541--kubernetes-io-master-staging.netlify.app/docs/concepts/services-networking/service/#defining-a-service

File: content/en/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -74,8 +74,8 @@ a new instance.
 The name of a Service object must be a valid
 [DNS label name](/docs/concepts/overview/working-with-objects/names#dns-label-names).
 
-For example, suppose you have a set of Pods that each listen on TCP port 9376
-and carry a label `app=MyApp`:
+For example, suppose you have a set of Pods and each of them listens on TCP port 9376
```
Comment (kbhawkey) [inline content/en/docs/concepts/services-networking/service.md:77]: Hi @hezhizhen . Looking better.
For example, suppose you have a set of Pods where each listens on TCP port 9376 and contains a label ...

Comment (hezhizhen) [reply content/en/docs/concepts/services-networking/service.md:77]: sounds good.

Comment (hezhizhen) [reply content/en/docs/concepts/services-networking/service.md]: @kbhawkey Take another look?

Comment (kbhawkey) [reply content/en/docs/concepts/services-networking/service.md]: 👀

**Comment by kbhawkey** [comment]
Thanks for contributing @hezhizhen !
/lgtm

**Pull request merged**